src/views/scans/showScansModal.tsx

```diff
@@ -38,29 +84,10 @@ const ShowScansModal: React.FC<ShowScansModalProps> = ({
   const [sortedScanJobs, setSortedScanJobs] = useState(scanJobs);
 
   useEffect(() => {
-    setSortedScanJobs(scanJobs);
-  }, [scanJobs]);
-
-  const getSortValue = (job: Pick<ScanJobType, 'start_time' | 'end_time' | 'status'>, index: number) => {
-    if (index === 0) {
-      return new Date(job.end_time || job.start_time).getTime();
-    }
-    if (index === 1) {
-      return job.status;
+    if (scanJobs.length > 0) {
+      setSortedScanJobs(sortByLatestTime(scanJobs));
```

Comment:
@cdcabrera After applying all the changes requested, the tests related to this modal were never finishing. I was running  into "Maximum component depth exceeded". This seemed to be causing infinite loops, due to re-rendering of parent component:
```
useEffect(() => {
  const initialSortedJobs = sortByLatestTime(scanJobs);
  setSortedScanJobs(initialSortedJobs);
}, [scanJobs]);
```
The solution I have now solves the problem, let me know if it was the best path forward.